### DISCUSSION TITLE
How to export transcripts as .srt files?

### DISCUSSION THREAD

**Discussion by The3IC** [intel/openvino-plugins-ai-audacity#166] [Q&A] [answered]
I was expecting to find srt export under the Audacity File -> Export?

**Comment by RyanMetcalfeInt8** [comment]
Hi @The3IC,

I'm not sure -- and I agree that this would be a useful feature, but it seems like this question / feature request is more appropriately directed at Audacity project itself (Instead of these AI plugins)? Would you mind re-filing this one over there? (https://github.com/audacity/audacity/issues)

**Comment by The3IC** [comment]
OK, but quite frankly, what is the purpose of generating the transcripts unless you can export them (to be used in videos, eg on youtube)? Can't see how they are useful (only) inside Audacity?

**Comment by RyanMetcalfeInt8** [comment]
> OK, but quite frankly, what is the purpose of generating the transcripts unless you can export them (to be used in videos, eg on youtube)? Can't see how they are useful (only) inside Audacity?

Just one example -- For podcasters, I've heard that the label track is useful as a guide / companion to the audio track. You can find the audio segment you're looking for much quicker if it's labeled.

And like I said, I do agree -- but I don't see an 'export SRT' as a feature that we are going to implement / provide with these plugins.

A quick google of 'Audacity export srt' shows many leads for how you might be able to do this.

**Comment by The3IC** [comment]
Ah, yes, found a few extensions for .srt generation from labels, now that you mention it. For anybody else, with the same question, I can report back here a bit later if you don't close this quite yet.

**Comment by The3IC** [comment, marked as answer]
OK, I have the simple answer, .srt export is a standard feature of Audacity BUT it is not "on" by default. Instead it needs to be turned on in Edit -> Preferences -> Import / Export Preferences in the Exported Label Style box if you wish or need to use this. 

There is one downside, if you have generated both the original language and a translation, both are put into the same file.

So all good..

**Comment by RyanMetcalfeInt8** [comment]
Hi @The3IC,

Very good to know, thanks for looking into this. Instead of closing this issue, which I think will make it more difficult for others to find, I'll convert it to a discussion.
